### PR TITLE
[#16] Support GHC-8.8.2

### LINKED ISSUE #16: Support GHC-8.8.1
TODO:
- [ ] Update Travis config
- [ ] Add `-Wmissing-deriving-strategies` to `.cabal` file ghc options

### LINKED ISSUE #23: Move CI to GitHub Actions

### PR DESCRIPTION
**Pull request by vrom911**
Resolves #16
Resolves #23

### CODE REVIEW

File: hakyll-shortcut-links.cabal [hunk 1]
```diff
@@ -9,31 +9,39 @@ license:             MPL-2.0
 license-file:        LICENSE
 author:              Veronika Romashkina, Dmitrii Kovanikov
 maintainer:          Kowainik <xrom.xkov@gmail.com>
-copyright:           2019 Kowainik
+copyright:           2019-2020 Kowainik
```
Comment (chshersh) [inline hakyll-shortcut-links.cabal:12]: Just noticed: let's also update years in module headers

**Comment by vrom911** [comment]
Travis will never manage to build this project, will it?

I will make #23 inside this PR, so we can have some working CI at least. :disappointed:

**Comment by chshersh** [comment]
@vrom911 I agree with you plan. It looks like GItHub Actions are noticeably faster than Travis CI and have higher delays on the output. So I'm okay with switching to GitHub Actions in this PR.

**Review by chshersh — APPROVED** [review]
Looks great! 👍 😍

**Pull request merged**